<?xml version="1.0"?>

<launch>

	<node pkg="holodeck_controller" type="controller_node" name = "controller_node" output = "screen" >
		<remap from="video" to="holodeck/image" />
		<remap from="command" to="holodeck/command"/>
		<remap from="state" to="holodeck/state"/>
	</node>

<!-- 	<node pkg="joy" type="joy_node" name="joy_node" output = "screen" >
		<remap from="joy" to="holodeck/joy"/>
		<param name="autorepeat_rate" value="30"/>
	</node> -->

	<node pkg="ros_holodeck" type="ros_holodeck_node" name ="ros_holodeck_node" output="screen"/>

	<node pkg="keyboard_input" type="keyboard_input_node" name="keyboard_input_node" output="screen" launch-prefix="xterm -e"/>




		<!-- <node pkg="rqt_image_view" type="rqt_image_view" name="rqt_image_view" /> -->
		

	 	<!-- <node pkg="rqt_reconfigure" type="rqt_reconfigure" name="dynamic_reconfigure" /> -->




</launch>